src/Engine/Manager/ProfilingManager.cpp

```diff
@@ -118,7 +118,7 @@ void ProfilingManager::ShowResult(Result& result) {
     
     ImGui::NextColumn();
     if (result.parent != nullptr) {
-        ImGui::ProgressBar(result.duration / result.parent->duration, ImVec2(0.0f,0.0f));
+        ImGui::ProgressBar(result.duration / result.parent->duration, ImVec2(0.0f, 0.0f));
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') comma should be followed by whitespace [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT009)